sdk/python/feast/infra/registry/sql.py

```diff
@@ -284,15 +284,17 @@ def __init__(
         )
 
     def _sync_feast_metadata_to_projects_table(self):
-        feast_metadata_projects: set = []
+        feast_metadata_projects: dict = {}
         projects_set: set = []
         with self.read_engine.begin() as conn:
             stmt = select(feast_metadata).where(
                 feast_metadata.c.metadata_key == FeastMetadataKeys.PROJECT_UUID.value
             )
             rows = conn.execute(stmt).all()
             for row in rows:
-                feast_metadata_projects.append(row._mapping["project_id"])
+                feast_metadata_projects[row._mapping["project_id"]] = int(
+                    row._mapping["last_updated_timestamp"]
+                )
```

Comment:
Can you explain what this is doing? I don't really follow

Reply:
Feast is deprecating feast_metadata table in the favor of projects. We are doing a sync of projects from feast_metadata to projects table. Earlier we are creating projects but missing the create timestamp from feast_metadata to projects. It creates inconsistencies in registry. This fix avoids that inconsistency.